pcap/search.go

```diff
@@ -147,41 +148,27 @@ type SearchReader struct {
 	reader pcapio.Reader
 	opts   gopacket.DecodeOptions
 	npkt   int
+	buf    []byte
 }
 
-func (s *Search) Reader(r pcapio.Reader) *SearchReader {
+func (s *Search) Reader(r pcapio.Reader) io.Reader {
```

Comment:
This can still return a SearchReader; there's no reason to limit its method set, since SearchReader is exported.

```suggestion
func (s *Search) Reader(r pcapio.Reader) *SearchReader {
```

Reply:
ok